tests/testthat/test-modules.R

```diff
@@ -125,6 +125,13 @@ testthat::test_that("module requires filters argument to be a character", {
   testthat::expect_error(module(filters = NA_character_), "Contains missing values")
 })
 
+testthat::test_that("module shouldn't have filter = NULL and data or datasets in the formals", {
```

Comment:
```suggestion
testthat::test_that("module shouldn't have filter = NULL and data in the formals", {
```

Reply:
Sorry, I realized to late about this suggestion